docker/ckan-dev-settings.ini

```diff
@@ -136,6 +136,9 @@ ckan.views.default_views = image_view text_view recline_view
 # Customize which image formats the image_view plugin will show
 #ckan.preview.image_formats = png jpeg jpg gif
 
+#harvest settings
+ckan.harvest.mq.type = redis
+
```

Comment:
We also need additional modifications here to reflect the fact that our redis server is coming from elsewhere (the default being `localhost`, which is not correct in the docker-compose case)